doc/adrs/013_improve_edge_source_and_target_anchors.adoc

```diff
@@ -0,0 +1,39 @@
+= ADR-013 - Improve the source and target edge anchors
+
+== Context
```

Comment:
With asciidoc, ALWAYS use one sentence per line (it's on purpose to help with Git and others VCS).

Reply:
done